stdlib/public/core/Optional.swift

```diff
@@ -232,7 +232,7 @@ extension Optional where Wrapped: ~Copyable {
   ) throws(E) -> U? {
     #if $NoncopyableGenerics
     switch self {
-    case .some(_borrowing y):
+    case .some(borrowing y):
```

Comment:
I believe this syntax is not happening with the recently accepted proposal if you want to have to update this again in a future patch.

Reply:
https://github.com/apple/swift-evolution/commit/aa266d2f7d95581a82d2ba983ca637a1db4e83bd